pkg/auth/authTokensGet_test.go

```diff
@@ -121,10 +148,69 @@ func TestInvalidPathReturnsError(t *testing.T) {
 	console := utils.NewMockConsole()
 
 	//When
-	err := GetTokens(apiClient, console)
+	err := GetTokens(apiClient, console, "")
 
 	//Then
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "GAL1146E")
 	assert.Contains(t, err.Error(), "Could not get list of tokens from API server")
 }
+
+func TestMissingLoginIdFlagReturnsBadRequest(t *testing.T) {
+	//Given...
+	serverState := "missingLoginId"
+	server := NewAuthTokensServletMock(t, serverState)
+	apiClient := api.InitialiseAPI(server.URL)
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `GAL1155E: The id provided by the --id field cannot be an empty string.`
+
+	//When
+	err := GetTokens(apiClient, console, "   ")
+
+	//Then
+	assert.NotNil(t, err)
+	assert.Equal(t, expectedOutput, err.Error())
+}
+
+func TestLoginIdWithSpacesReturnsBadRequest(t *testing.T) {
+	//Given...
+	serverState := "invalidLoginIdFlag"
+	server := NewAuthTokensServletMock(t, serverState)
+	apiClient := api.InitialiseAPI(server.URL)
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `GAL1157E: 'galasa admin' is not supported as a valid value. Valid value should not contain spaces. A value of 'admin' is valid but 'galasa admin' is not.`
+
+	//When
+	err := GetTokens(apiClient, console, "galasa admin")
+
+	//Then
+	assert.NotNil(t, err)
+	assert.Equal(t, expectedOutput, err.Error())
+}
+
+func TestGetTokensByLoginIdReturnsOK(t *testing.T) {
+	//Given...
+	serverState := "populatedByLoginId"
+	server := NewAuthTokensServletMock(t, serverState)
+	apiClient := api.InitialiseAPI(server.URL)
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `tokenid                   created(YYYY-MM-DD) user     description
+098234980123-1283182389   2023-12-03          mcobbett So I can access ecosystem1 from my laptop.
+8218971d287s1-dhj32er2323 2024-03-03          mcobbett Automated build of example repo can change CPS properties
+
+Total:2
+`
+
+	//When
+	err := GetTokens(apiClient, console, "mcobbett")
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
```

Comment:
Alongside your unit tests, it would be good to add tests similar to the ones in `runs-tests.sh` to exercise the `auth tokens get` against a running ecosystem - happy to talk through that if needed!